.github/workflows/chez-build.yml

```diff
@@ -36,8 +36,6 @@ jobs:
           fetch-depth: 50
       - name: Download pb boot files
         run: make fetch-pb
-      - name: Proceed with TARGET_MACHINE == ${{ matrix.mach }}
```

Comment:
@pmatos

Can you confirm that this step can be safely removed? It looks like `TARGET_MACHINE` is an uninitialized variable, so it's always empty.

Reply:
Yes, that can be removed. I think that was debug only and somehow got committed.

Reply:
Somehow my previous comments don't show up here. I think I commented in the wrong place. Yes, this can be removed.